JSONFormat.go

```diff
@@ -39,7 +45,7 @@ func (j jsonEntry) MarshalJSON() ([]byte, error) {
 		return nil, ErrNoMessage
 	}
 
-	_, err := buf.WriteString(fmt.Sprintf("{\"%s\":\"%s\", \"%s\":\"%s\", \"%s\":\"%s\"", j.levelKey, j.Level, j.eventKey, j.Event, j.messageKey, j.Message))
+	_, err := buf.WriteString(fmt.Sprintf("{\"%s\":\"%s\", \"%s\":\"%s\", \"%s\":\"%s\"", j.levelKey, j.Level, j.eventKey, j.Event, j.messageKey, escapeSpecialChars(j.Message)))
```

Comment:
Hey @GVdP-Travix, (@fahad19 mentioned you worked on this part)

Was there any particular reason to implement the json serialization on such low level? Can't we use some sort of library to construct the json object, which would take care of escaping the special chars for us?